src/saml20_clj/sp/request.clj

```diff
@@ -3,9 +3,17 @@
             [java-time.api :as t]
             [ring.util.codec :as codec]
             [saml20-clj.coerce :as coerce]
-            [saml20-clj.crypto :as crypto]
             [saml20-clj.encode-decode :as encode-decode]
-            [saml20-clj.state :as state]))
+            [saml20-clj.state :as state])
+  (:import org.opensaml.messaging.context.MessageContext
+           [org.opensaml.saml.common.messaging.context SAMLBindingContext SAMLEndpointContext SAMLPeerEntityContext]
+           org.opensaml.saml.common.xml.SAMLConstants
+           org.opensaml.saml.saml2.binding.encoding.impl.HTTPRedirectDeflateEncoder
+           [org.opensaml.saml.saml2.core AuthnRequest NameIDType]
+           [org.opensaml.saml.saml2.core.impl AuthnRequestBuilder IssuerBuilder NameIDPolicyBuilder]
+           org.opensaml.saml.saml2.metadata.impl.SingleSignOnServiceBuilder
+           org.opensaml.xmlsec.context.SecurityParametersContext
+           org.opensaml.xmlsec.SignatureSigningParameters))
```

Comment:
We don't have the linter for this installed in this library, what do you think of adding 
`(set! *warn-on-reflection* true)` after the ns form? That would help us be more confident that we are doing the type hinting correction.

Reply:
It is not blocking, but it would be a decent dev-chore to copy over some of our kondo config. In this case I was talking about the [warn-on-reflection](https://github.com/metabase/metabase/blob/master/.clj-kondo/config.edn#L33) linter.